dexbot/strategies/base.py

```diff
@@ -154,7 +154,7 @@ def configure(cls, return_base_config=True):
                           'BitShares account name for the bot to operate with',
                           ''),
             ConfigElement('market', 'string', 'USD:BTS', 'Market',
```

Comment:
Default value USD:BTS should be changed too.

Reply:
make it a separate issue please.

Reply:
Why separate? This change is related to your change. You changed ordering in one place, so change the default too to make your change consistent.

Reply:
what do you want USD:BTS to be changed to? BTS:USD or BTS/USD? Please be specific.

Reply:
To BTS/USD

Reply:
ok done.